### PR TITLE
Few fixes

### PR DESCRIPTION
**Pull request by KipK**
some minor changes.
- Doc updates
- Updated `shaper` type to match other modules
- a few fixes found while testing OpenEVSE/openevse_wifi_gui#87

### CODE REVIEW

File: docs/user-guide.md [hunk 1]
```diff
@@ -88,15 +96,20 @@ A [OpenEnergyMonitor Solar PV Energy Monitor](https://guide.openenergymonitor.or
...
 ### Eco Mode Setup
 
+#### using MQTT
 * Enable MQTT Service
 * [emonPi MQTT credentials](https://guide.openenergymonitor.org/technical/credentials/#mqtt) should be pre-populated
 * Enter solar PV generation or Grid (+I/-E) MQTT topic e.g. Assuming [standard emonPi Solar PV setup](https://guide.openenergymonitor.org/applications/solar-pv/), the default MQTT feeds are:
   * Grid Import (positive Import / Negative export*): `emon/emonpi/power1`
   * Solar PV generation (always postive): `emon/emonpi/power2`
 
+#### using HTTP
```
Comment (jeremypoulter) [inline docs/user-guide.md:106]: Can you also update `api.yml` ?

File: src/web_server.cpp [hunk 2]
```diff
@@ -709,7 +709,7 @@ handleScheduleGet(MongooseHttpServerRequest *request, MongooseHttpServerResponse
     response->setCode(200);
     serializeJson(doc, *response);
   } else {
-    response->setCode(404);
+    response->setCode(200);
```
Comment (jeremypoulter) [inline src/web_server.cpp:712]: The `404` is correct in this case, the requested event does not exist and the `404` allows you to detect this

File: src/web_server.cpp [hunk 3]
```diff
@@ -811,7 +811,7 @@ void handleOverrideGet(MongooseHttpServerRequest *request, MongooseHttpServerRes
     manual.getProperties(props);
     props.serialize(response);
   } else {
-    response->setCode(404);
+    response->setCode(200);
```
Comment (jeremypoulter) [inline src/web_server.cpp:814]: Same as above, the `404` is correct

File: src/web_server_claims.cpp [hunk 4]
```diff
@@ -29,7 +29,7 @@ handleEvseClaimsGet(MongooseHttpServerRequest *request, MongooseHttpServerRespon
     response->setCode(200);
     serializeJson(doc, *response);
   } else {
-    response->setCode(404);
+    response->setCode(200);
```
Comment (jeremypoulter) [inline src/web_server_claims.cpp:32]: Same as above the `404` is correct

**Comment by jeremypoulter** [comment]
`/claims` returns 200 with an empty array when no claims are set. We can also add a `claim_version` to the status, same as the schedule if you need an event to update the claims

File: src/web_server.cpp (hunk 3 above)
Comment (KipK) [reply src/web_server.cpp:814]: I think the REST API should only answer 404 for not found element like /claims/{id}  /schedule/{id}
But not for getting claims/ or /override as it is still a real answer with needed info ( array is empty )

**Comment by KipK** [comment]
> `/claims` returns 200 with an empty array when no claims are set. We can also add a `claim_version` to the status, same as the schedule if you need an event to update the claims, but the `404` is expected and the correct response in those cases.

ho I see, I'm calling /claims/{id} from the UI to only get the claim from Manual clientid.
I can solve that by grabing the whole package.

**Comment by KipK** [comment]
a claim_revision in status could be usefull yes

**Review by jeremypoulter — APPROVED** [review]

**Pull request merged**